src/shared/state/Banners/Banners.schema.ts

```diff
@@ -1,13 +1,16 @@
-import { AppletWithoutChangesBanner } from 'modules/Builder/components/Banners';
-import { VersionWarningBanner } from 'shared/components/Banners/VersionWarningBanner';
+import { BannerProps } from 'shared/components/Banners/Banner/Banner.types';
 
-export const BannerComponents = {
+export enum BannerType {
```

Comment:
Not a blocking comment, but is the expectation that with any new banner added to the site, we would add a new type here? I would expect these banner types to be much more generic (`SuccessBanner`, `WarningBanner`, `ErrorBanner`, etc) with the children supplied via `bannerProps`, and not being much more opinionated than that...

...but then again, those wrapper components also wouldn't be doing much more than pre-configuring some values for `severity` `duration`. Anyway, just food for thought. My PRs were going to reuse the `SaveSuccessBanner` for save success and duplicate success

Reply:
I've also been reusing `SaveSuccessBanner` for most success situations. You're right that it is basically just props that are changing (with `children` being one of those props), but I suppose the benefit of having distinct components is cases where the content is more complex and that banner is called from multiple locations, just to offer a level of reusability. (Which, as I say that, it now occurs to me I probably should have done with the "transfer ownership" banner! Maybe I'll make a change there…)

Reply:
Got it, that seems reasonable!